akka-stream-tests/src/test/scala/akka/stream/scaladsl/FlowConflateSpec.scala

```diff
@@ -168,6 +169,9 @@ class FlowConflateSpec extends AkkaSpec {
       sub.expectRequest(1)
       sub.sendNext(3)
 
+      // delay requesting to avoid race condition
```

Comment:
What race condition? If there is one, then sleeping 100ms will not be good enough (CI builds run on weird systems).